egs2/wsj0-2mix/enh1/conf/pitch.conf

```diff
@@ -0,0 +1 @@
+--sample-frequency=16000
```

Comment:
I don't recommend to use the pitch feature. We cannot easily integrate it with ASR. Also, it is not Kaldi-free

Reply:
Removed.